scripts/homepage.js

```diff
@@ -68,6 +68,11 @@ function process_payment (amount, token) {
     payment_http.send('amount=' + amount + '&token=' + token.id);
 }
 
+function payment_complete (amount) {
```

Comment:
Would it not be better to set the cookie from the stripe payment hook we have serverside? We have that http request anyway, and setting the cookie there seems more logical to me.